financial-connections-core/src/main/java/com/stripe/android/financialconnections/utils/HostedAuthUrlBuilder.kt

```diff
@@ -1,23 +1,22 @@
-package com.stripe.android.financialconnections
+package com.stripe.android.financialconnections.utils
 
+import androidx.annotation.RestrictTo
 import com.stripe.android.core.networking.QueryStringFactory
-import com.stripe.android.financialconnections.ElementsSessionContext.BillingDetails
-import com.stripe.android.financialconnections.ElementsSessionContext.PrefillDetails
+import com.stripe.android.financialconnections.ElementsSessionContext
 import com.stripe.android.financialconnections.launcher.FinancialConnectionsSheetActivityArgs
-import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest
-import com.stripe.android.financialconnections.utils.toApiParams
 import com.stripe.android.model.IncentiveEligibilitySession
 import com.stripe.android.model.LinkMode
 
-internal object HostedAuthUrlBuilder {
+@RestrictTo(RestrictTo.Scope.LIBRARY)
+object HostedAuthUrlBuilder {
 
     fun create(
```

Comment:
@mats-stripe success / cancel redirection started working when moving to use the URL builder we use in FC full.